ansible/roles/daemon/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Install packages required by daemon, if necessary
+  become: true
+  become_method: sudo
+  dnf:
+      name: "{{ item }}"
+      state: present
+  with_items: "{{ daemon.packages }}"
+
+- name: Use go get to download leapp daemon and dependencies
+  raw: "go get {{ daemon.repo }}/..."
```

Comment:
most of the code is available in ~/leapp, I would recommend to create a symlink there to the $GOPATH/src/..../leapp-go so all source codes will be available from single point of entry.

Reply:
Also, I would suggest to add packages like goimports in order to make the tests work